omero/sysadmins/omeroweb-upgrade.rst

```diff
@@ -147,7 +146,7 @@ Restart OMERO.web
 ^^^^^^^^^^^^^^^^^
 
 -  If anything goes wrong, please send the output of
```

Comment:
Why is this under a "Restart OMERO.web heading" rather than a separate "Diagnostics" heading?  These seem like separate activities.  Also, having OMERO.web in the heading is redundant since the whole section is about that.

Reply:
Because JM cut & pasted content from the server upgrade page basically. Another round of cleanup is probably due there but without the scope of this PR.